lib/tests/test_conversion.py

```diff
@@ -84,7 +89,7 @@ def test_transform_coordinates_to_esri54009(from_epsg, from_x, from_y, to_x, to_
     assert math.isclose(x, to_x, abs_tol=0.1) # tolerance 0.1m
     assert math.isclose(y, to_y, abs_tol=0.1)
 
-
+@pytest.mark.filterwarnings("ignore::FutureWarning")
```

Comment:
I don't think we should ignore these warnings. They may be annoying, but instead of ignoring them, we should resolve them. Not in this PR of course.

If these warnings are such an intolerable annoyance to you at this point, add them to a central pytest.ini instead of cluttering the tests. That pytest.init must have a clear `FIXME` comment.

Reply:
Having them directly above the test in which they happen makes it much easier to come back to and clean them up later, but I also agree we shouldn't be ignoring them in the longterm.